types/chroma-js/index.d.ts

```diff
@@ -321,7 +321,7 @@ declare namespace chroma {
          * Set out format for scale() call. Passing null will result in a scale which outputs colors.
          */
         out(format: null): Scale;
-        out<K extends keyof ColorSpaces>(format: K): Scale<ColorSpaces[K]>;
+        out<K extends keyof ColorSpaces>(format: K | 'hex'): Scale<ColorSpaces[K] | string>;
```

Comment:
Would it make more sense to add a separate overload `out(format: 'hex'): Scale<string>`?  That would tie the parameter and return types together more closely.

Reply:
Ah yes - good idea, thanks. I've added a new commit with the suggested changes.